cypress/features/regression/designSystem/textEditor.feature

```diff
@@ -0,0 +1,99 @@
+Feature: Design System Text Editor component
+  I want to test Design System Text Editor component
+
+  Background: Open Design System Text Editor component page
+    Given I open design systems basic "Text Editor" component in no iframe
+
+  @positive
```

Comment:
Maybe we should add TC that verify if we can type more that character limit ?

Reply:
added